src/openforms/plugins/plugin.py

```diff
@@ -0,0 +1,44 @@
+from typing import List, Tuple
+
+from django.utils.translation import gettext_lazy as _
+
+
+class AbstractPlugin:
```

Comment:
naming suggestions welcome :shrug:

Reply:
`AbstractBasePlugin` is fine, similar to Django's `AbstractBaseUser` class